File: Beecrowd/Beginner/2003 - 2176/2031 - Rock, Paper, Airstrike/C++/2031_RockPaperAirstrike.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int n;
    cin >> n;

    for (int i = 0; i < n; i++) {
        string player1, player2;
        cin >> player1 >> player2;

        if (player1 == "ataque" && player2 == "pedra") cout << "Jogador 1 venceu" << endl;
        else if (player1 == "pedra" && player2 == "ataque") cout << "Jogador 2 venceu" << endl;
        else if (player1 == "pedra" && player2 == "papel") cout << "Jogador 1 venceu" << endl;
        else if (player1 == "papel" && player2 == "pedra") cout << "Jogador 2 venceu" << endl;
        else if (player1 == "papel" && player2 == "ataque") cout << "Jogador 2 venceu" << endl;
        else if (player1 == "ataque" && player2 == "papel") cout << "Jogador 1 venceu" << endl;
        else if (player1 == "papel" && player2 == "papel") cout << "Ambos venceram" << endl;
        else if (player1 == "pedra" && player2 == "pedra") cout << "Sem ganhador" << endl;
        else if (player1 == "ataque" && player2 == "ataque") cout << "Aniquilacao mutua" << endl;
    }

    return 0;
}

File: Beecrowd/Beginner/2003 - 2176/2031 - Rock, Paper, Airstrike/Python/2031_RockPaperAirstrike.py
test_cases = int(input())

for _ in range(test_cases):
    player1 = input()
    player2 = input()

    if player1 == "ataque" and player2 == "pedra":
        print("Jogador 1 venceu")
    elif player1 == "pedra" and player2 == "ataque":
        print("Jogador 2 venceu")
    elif player1 == "pedra" and player2 == "papel":
        print("Jogador 1 venceu")
    elif player1 == "papel" and player2 == "pedra":
        print("Jogador 2 venceu")
    elif player1 == "papel" and player2 == "ataque":
        print("Jogador 2 venceu")
    elif player1 == "ataque" and player2 == "papel":
        print("Jogador 1 venceu")
    elif player1 == "papel" and player2 == "papel":
        print("Ambos venceram")
    elif player1 == "pedra" and player2 == "pedra":
        print("Sem ganhador")
    elif player1 == "ataque" and player2 == "ataque":
        print("Aniquilacao mutua")
